packages/webapp/pages/posts/[id]/edit.tsx

```diff
@@ -46,9 +49,13 @@ function EditPost(): ReactElement {
     onMutate: () => {
       onAskConfirmation(false);
     },
-    onSuccess: async () => {
+    onSuccess: async (data) => {
       clearDraft();
-      await push(post.commentsPermalink);
+      await push(data.commentsPermalink);
```

Comment:
New data is returned after edit so we can redirect to new link with slug. Before it would be 404 if you changed the post title.